test/e2e/framework/framework.go

```diff
@@ -40,7 +41,7 @@ func ResetClusterProxyConfig(client *ClientSet) error {
 }
 
 func DeleteAll(t *testing.T, client *ClientSet) {
-	resources := []string{"Deployment", "Service", "Route", "ConfigMap"}
+	resources := []string{"Deployment", "Service", "Route", "ConfigMap", "OCCLIDownloads", "ODOCLIDownloads"}
```

Comment:
Pondering this.

Reply:
A fix could be, if we don't want to restructure the functions:

```golang
// use the actual names & run the switch against that, so we don't have a string representing another string:
resources := []string{
   "Deployment",   // might want to update to "console-deployment"
   "Service",  // might want to update to "console-service"
   // etc
   consoleapi.OCCLIDownloadsCustomResourceName,
   consoleapi.ODOCLIDownloadsCustomResourceName,
```
And maybe just leave a `TODO: ideally use Name + Kind` as something we could work on later if it became an issue.